src/python/pants/backend/javascript/package/rules_integration_test.py

```diff
@@ -122,8 +124,8 @@ def configure_runner_with_js_project(request, rule_runner: RuleRunner) -> RuleRu
                     "scripts": {
                         "build": "parcel build index.mjs --dist-dir=dist --cache-dir=.parcel-cache"
                     },
-                    "dependencies": {"child-lib": "*"},
-                    "devDependencies": {"parcel": "^2.12.0"},
+                    "dependencies": {"child-lib": child_dep},
```

Comment:
I needed to do this, because pnpm/yarn/npm don't all seem to like the same specification of workspace dependencies (anymore?)